README.md

```diff
@@ -63,7 +63,7 @@ Creating an editor using a CKEditor 5 build is very simple and can be described
 In your HTML page add an element that CKEditor should replace:
 
 ```html
-<textarea name="content" id="editor"></textarea>
+<div name="content" id="editor"></div>
```

Comment:
`div` tag can not contain a `name` attribute. This issue is repeated throughout rest of the PR.

Reply:
I was on drugs yesterday, sorry :( Fixing it together with https://github.com/ckeditor/ckeditor5/issues/1806#issuecomment-559699107 as it makes sense to clarify this issue.